src/devtools/client/debugger/src/components/shared/Modal.tsx

```diff
@@ -10,35 +10,42 @@ import { Transition } from "react-transition-group";
 
 export const transitionTimeout = 50;
 
-export class Modal extends React.Component {
-  onClick = e => {
-    e.stopPropagation();
-  };
+interface IModalProps {
+  additionalClass: string;
+  children: React.ReactNode;
+  handleClose: () => void;
+  status: string;
+  width: string | number;
+}
 
-  render() {
-    const { additionalClass, children, handleClose, status, width } = this.props;
+interface ISlideProps extends IModalProps {
+  in: boolean | undefined;
+}
```

Comment:
@markerikson I tried to read the component API and the associated invocations but not entirely sure if I captured the types properly here.

Reply:
Looks like `IModalProps` doesn't actually have a `status` field or it needs to be optional, based on the TS error from the build step